pkgs/by-name/au/authentik/radius.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildGoModule, authentik }:
+
+buildGoModule {
+  pname = "authentik-radius-outpost";
+  inherit (authentik) version src;
+
+  vendorHash = "sha256-UIJBCTq7AJGUDIlZtJaWCovyxlMPzj2BCJQqthyb1z4=";
+
+  CGO_ENABLED = 0;
+
+  subPackages = [ "cmd/radius" ];
+
+  meta = authentik.meta // {
+    description = "The authentik radius outpost. Needed for the extendal ldap API.";
```

Comment:
I don't think radius provides the external ldap api

Reply:
thx, fixed